travelpal/assets/js/components/Search.jsx

```diff
@@ -1,34 +1,11 @@
 import React from 'react';
-import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { FormGroup, Label, Input } from 'reactstrap';
-import SearchResults from './SearchResults';
 
-import api from '../api';
 
 // Renders the search page
-export default function Search({ userId, users, search, updateSearch }) {
-
-  // Updates the form with the query
-  function update(ev) {
-    const query = $(ev.target).val().toLowerCase();
-    updateSearch(query);
-  }
+export default function Search(props) {
 
   return (
-    <div className="page-content">
-      <FormGroup>
-        <Label>Search for Friends</Label>
-        <Input onChange={update} value={search} type="search" name="search"
-          placeholder="Enter name or username" />
-      </FormGroup>
-      <SearchResults userId={userId} users={users} search={search} />
-    </div>
+    <div>Search</div>
```

Comment:
Should return
```
<div className="page-content">
    Search
</div>
```
because we need that `className` for things to align properly.

Reply:
https://github.com/kimberlypn/TravelPal/pull/29/commits/7af2c5f1c0b310fa1f005f4d7876347e0bac6988